src/Stratis.Bitcoin.Features.SignalR/EventsHub.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Extensions.Logging;
+
+namespace Stratis.Bitcoin.Features.SignalR
+{
+    public class EventsHub : Hub
+    {
+        private readonly ILogger<EventsHub> logger;
+
+        public EventsHub(ILoggerFactory loggerFactory)
+        {
+            this.logger = loggerFactory.CreateLogger<EventsHub>();
+        }
+
+        public override Task OnConnectedAsync()
+        {
+            this.logger.LogDebug("New client with id {id} connected", this.Context.ConnectionId);
```

Comment:
Dont use string interpolation

Reply:
Also what is `id`? Shouldnt it be 0 in this case?

Reply:
@fassadlr ?

Reply:
@fassadlr see my reply above for the same comment. This is not a string interpolation (note no $). Id would a GUID of the signalr connection id representing a client

Reply:
How does this work @fenix2222 ? I string.Format requires an integer here?